src/main/java/calculator/Calculator.java

```diff
@@ -0,0 +1,67 @@
+package calculator;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Calculator {
+	public String[] splitStr(String calStr){
+		return calStr.split(" ");
+	}
+
+	public boolean isOp(String op){
+		if("+".equals(op) || "-".equals(op) || "*".equals(op) || "/".equals(op)){
+			return true;
+		}
+		return false;
+	}
+
+	public int partCalculate(int x, String op, int y){
+		if("+".equals(op)){
+			return sum(x, y);
+		}
+		else if("-".equals(op)){
+			return sub(x, y);
+		}
+		else if("*".equals(op)){
+			return mul(x, y);
+		}
+		return div(x, y);
+	}
+
+	public int opNumSplitSave(String[] calStr){
+		List<String> opList = Arrays.stream(calStr)
+			.filter(s -> isOp(s))
```

Comment:
```suggestion
			.filter(this::isOp)
```

메서드 레퍼런스를 사용할 수 있습니다. `Effective Java, item 43. 람다보다는 메서드 참조를 사용하라` 를 참고하시면 좋습니다.